core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -239,6 +239,36 @@ public void completeCreateView() {
     assertThat(catalog().viewExists(identifier)).as("View should not exist").isFalse();
   }
 
+  @Test
+  public void createViewWithCustomMetadataLocation() {
```

Comment:
can you also please add a test to `TestViews` with a custom metadata location. You should be able to do a `DESCRIBE` in that test and verify that the metadata locations match

Reply:
Thank you! Added the test